cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuilder.java

```diff
@@ -82,8 +83,8 @@ public SlangBuildResults buildSlangContent(String projectPath, String contentPat
         return compiledSlangFiles;
     }
 
-    private RunTestsResults runTests(Map<String, Executable> contentSlangModels,
-                          String projectPath, String testsPath, List<String> testSuites){
+    IRunTestResults runTests(Map<String, Executable> contentSlangModels,
```

Comment:
it makes sense to have the visibility package protected to be used from unit test, but also because it abstracts running of tests